src/main/java/org/tyutyunik/school/service/impl/AvatarServiceImpl.java

```diff
@@ -40,8 +42,11 @@ public AvatarServiceImpl(AvatarRepository avatarRepository,
         this.pathDir = pathDir;
     }
 
+    private static final Logger logger = LoggerFactory.getLogger(AvatarServiceImpl.class);
+
     @Override
     public long addAvatar(long studentId, MultipartFile multipartFile) {
+        logger.info("[INFO] [AvatarService] Was invoked addAvatar");
```

Comment:
INFO и AvatarService в логгере можно не указывать, логгер сам добавит имя и уровень
```suggestion
        logger.info("Was invoked addAvatar");
```